packages/input-mask/README.md

```diff
@@ -91,6 +91,34 @@ return (
 
 In most cases you'll want to use `onInput` and `onPaste`.
 
+
+## Showing the pattern during input
+
+The placeholder attribute is nice, but it cannot show the pattern during input. In order to rectify this shortcoming, this package has a `createMaskPattern` export:
+
+```tsx
+import { createInputMask, createMaskPattern } from "@solid-primitives/input-mask";
+
+return <>
+  <style>{`
+    input { border: 0.1rem solid currentColor; padding: 0.5rem 0.75rem; }
+    label[data-mask-value] { position: absolute; border: 0.1rem solid transparent; padding: 0.5rem 0.75rem; }
+    label[data-mask-value]::before { content: attr(data-mask-value); color: transparent; }
+    label[data-mask-pattern]::after { content: attr(data-mask-pattern); opacity: 0.7; }
+  `}</style>
+  <label for="datefield">ISO Date:</label>
+  <label for="datefield"></label>
```

Comment:
This label shouldn’t be duplicated, right?

Reply:
Yes, it should. One is for the descriptive text, the other one for the pattern. It's described in the README.md.

Reply:
Maybe I should add a descriptive id to make its function more obvious, though.